src/security/expect-ct.conf

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------
+# | Expect-CT                                                          |
+# ----------------------------------------------------------------------
+
+# Opt-in to Certificate Transparency (CT).
+#
+# Certificate Transparency helps guard against several types of certificate-
+# based threats, including misissued certificates, maliciously acquired
+# certificates, and rogue CAs (Certificate Authorities).
+#
+# (!) When a site requires Certificate Transparency (using the `enforce`
+#     directive), they are requesting that the browser check that any
+#     certificate for that site appears in public CT logs (1).
+#
+# It is advised (2) to only serve the Expect-CT header over a secure
+# connection.
+#
+# (1) https://www.certificate-transparency.org/known-logs
+# (2) https://httpwg.org/http-extensions/expect-ct.html#http-request-type
+#
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT
+# https://www.certificate-transparency.org/resources-for-site-owners
+# https://groups.google.com/forum/#!forum/certificate-transparency
+# https://httpwg.org/http-extensions/expect-ct.html
```

Comment:
When available, this link need to be updated to the IETF RFC.

Reply:
This does exist: https://tools.ietf.org/html/draft-ietf-httpbis-expect-ct, idk why I chose to include that version. Perhaps because - and quoting [Mark Nottingham](https://github.com/mnot/) "a more readable version". But yes, I can change to the ietf draft version instead. ^^

Reply:
Nah, I meant to the _FINAL_ version of the RFC! 👌😉